src/lib/mongoBackend/mongoRegistrationGet.cpp

```diff
@@ -93,11 +95,11 @@ static void setProvider(ngsiv2::Registration* regP, const mongo::BSONObj& r)
 
 /* ****************************************************************************
 *
-* setEntities - 
+* setEntities -
 */
-static void setEntities(ngsiv2::Registration* regP, const mongo::BSONObj& cr0)
+static void setEntities(ngsiv2::Registration* regP, const mongo::BSONObj& cr0, const char* dbLabel)
```

Comment:
Function is overparametriced, `dbLabel` is not needed at the field used for entities is not a parameter but fixed by the database model.

Reply:
Fixed in 5d851ef